chia/full_node/mempool_manager.py

```diff
@@ -184,13 +189,13 @@ def always(mm: MempoolManager, mi: MempoolItem) -> bool:
 
             item_inclusion_filter = always
 
-        log.info(f"Starting to make block, max cost: {self.constants.MAX_BLOCK_COST_CLVM}")
+        log.info(f"Starting to make block, max cost: {self.max_block_clvm_cost}")
         spend_bundles, cost_sum, additions, removals = self.process_mempool_items(item_inclusion_filter)
         if len(spend_bundles) == 0:
             return None
         log.info(
             f"Cumulative cost of block (real cost should be less) {cost_sum}. Proportion "
-            f"full: {cost_sum / self.constants.MAX_BLOCK_COST_CLVM}"
+            f"full: {cost_sum / self.max_block_clvm_cost}"
```

Comment:
I think it's important to restore this. People reading the log will assume that this metric is the same as it has always been

Reply:
Done.